packages/design-system/src/components/cookieTable/index.tsx

```diff
@@ -120,18 +129,30 @@ const CookieTable = forwardRef<
     [onRowClick]
   );
 
-  const selectedKey = useMemo(
-    () => Object.values(selectedFrameCookie ?? {})[0],
-    [selectedFrameCookie]
+  const selectedKey = useMemo(() => {
+    const key = Object.values(selectedFrameCookie ?? {})[0];
+
+    return key === null ? null : getCookieKey(key?.parsedCookie);
+  }, [selectedFrameCookie]);
+
+  const _conditionalTableRowClassesHandler = useCallback(
+    (row: TableRow, isRowFocused: boolean, rowIndex: number) => {
+      return conditionalTableRowClassesHandler(
+        row,
+        isRowFocused,
+        rowIndex,
+        selectedKey,
+        useIsBlockedToHighlight
```

Comment:
@mayan-000 could we rename this variable because at an initial glance without looking at the prop it looks like a hook?